src/component/Report.vue

```diff
@@ -1,27 +1,259 @@
+                    <td>{{item.lateWithPermission}}</td>
+                    <td>{{item.hourlyLeave}}</td>
+                    <td>{{item.replacementLeave}}</td>
+                    <td>{{item.noTapOutDay}}</td>
+                    <!--<td>{{item.transportFee}}</td>-->
+                </tr>
+
+                </tbody>
+            </table>
+        </section>
         <br/>
-        <report-table></report-table>
     </main>
 </template>
 <script>
-    import Filters from './report/FilterReport.vue';
-    import ReportTable from './report/TableReport.vue';
+
     export default {
         data() {
-            return {};
+            return {
+                startDate:'2017-01-01',
+                endDate:'2017-12-31',
+                selected:'All Department',
+                reportData: [
+                    { nik: '001', fullName: 'Ariel Christanto' , department:'Technology', daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+                    { nik: '002', fullName: 'Ariela Christanto' ,department:'Technology', daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+                    { nik: '003', fullName: 'Aimee Christanto' ,department:'Technology', daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+                    { nik: '004', fullName: 'Ariela Christanto' ,department:'Technology', daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+                    { nik: '005', fullName: 'Aeriel Christanto' ,department:'Technology', daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+                    { nik: '006', fullName: 'Arie Christanto' , department:'Technology',daysComing: 5, daysAbsence:6, sick:1,unpaidLeave:"-" ,yearlyLeave:'-',leaveEarly:'-', noTapOutDay:1,lateWithoutPermission:2,lateWithPermission:1,hourlyLeave:5,replacementLeave:9,noTapOut:2, transportFee:50000},
+
+                ]
+            };
         },
-        components:{Filters, ReportTable}
+
+        methods: {
+            showReport: function() {
+                this.$http.get('http://localhost:8080/report', {params:{
+                    dept:this.selected,
+                    startDate:this.mulai,
+                    endDate:this.selesai
+                }})
```

Comment:
`this.mulai` & `this.selesai`? Where does it come from? I don't see any assignment to both of those field